include/fastrtps/types/DynamicData.h

```diff
@@ -19,7 +19,7 @@
 #include <fastrtps/types/DynamicDataPtr.h>
 #include <fastrtps/types/DynamicTypePtr.h>
 
-//#define DYNAMIC_TYPES_CHECKING
+#define DYNAMIC_TYPES_CHECKING
```

Comment:
By default, types checking should be disabled, don't you think so?

Reply:
most definitely